uninstall.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * Plugin uninstaller logic.
+ *
+ * @package performance-lab
+ * @since 1.2.0
```

Comment:
This should technically be `n.e.x.t` per convention, but since the next release is going to be `1.2.0`, it's probably fine this time.

Reply:
Thank you